src/coprocessor/codec/mysql/decimal.rs

```diff
@@ -1463,6 +1544,10 @@ impl Decimal {
         }
         res
     }
+
+    pub fn is_zero(&self) -> bool {
+        !self.word_buf.iter().any(|&x| x != 0)
```

Comment:
It's ported from `tidb`(https://github.com/pingcap/tidb/blob/master/util/types/mydecimal.go#L1191),
And there seems no function `reset_to_zero`? @XuHuaiyu PTAL